pylint/checkers/classes/class_checker.py

```diff
@@ -525,6 +525,12 @@ def _has_same_layout_slots(
         "beginning with an underscore) is access outside the class or a "
         "descendant of the class where it's defined.",
     ),
+    "W0213": (
+        "Flag member %(overlap)s shares bit positions with %(sources)s",
```

Comment:
Great way to make the code more maintanable, I wonder if we should doit everywhere.

Reply:
Unfortunately it's not possible to express in a `pylint` rule (because it also involves `diff` context), but I think my approach would be:

If a string is being modified by a changeset _and_ contains `%`-replacement characters, then consider also changing the template to use named variables (or f-strings) during the same change -- although with some caution about larger potential refactoring if doing so would result in line-reformatting due to `black` / `flake8` rules.